src/commands/update/ip-blocks/put_ipblock_tag.go

```diff
@@ -0,0 +1,53 @@
+package ip_blocks
+
+import (
+	"github.com/spf13/cobra"
+	"phoenixnap.com/pnapctl/common/client/ip"
+	"phoenixnap.com/pnapctl/common/models/ipmodels"
+	"phoenixnap.com/pnapctl/common/printer"
+	"phoenixnap.com/pnapctl/common/utils"
+)
+
+var Filename string
+
+const commandName = "put ip-block tag"
+
+var PutIpBlockTagCmd = &cobra.Command{
+	Use:          "ip-block IP_BLOCK_ID",
+	Short:        "Updates a specific ip-block's tag.",
```

Comment:
Since we're allowing a list of tags, I would change this to `Updates an ip block's tags.`

Reply:
Done